### PR TITLE
build: activate source-terasology-modules plugin for testing

### PR DESCRIPTION
**Pull request by skaldarnar**
Based on #182, which fixes some issues with the plugin (would be failing otherwise).

- build(source-terasology-modules): active custom source module
- add data.json to gitignore

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -88,6 +88,12 @@ module.exports = {
         include_custom_fields: true,
       },
     },
+    {
+      resolve: "source-terasology-modules",
+      options: {
+        accessToken: process.env.GITHUB_TOKEN,
```
Comment (skaldarnar) [inline gatsby-config.js:94]:
```suggestion
        accessToken: process.env.MODULE_FETCH_GITHUB_TOKEN,
```

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "Terasology"
+    ]
+}
```
Comment (jdrueckert) [inline .vscode/settings.json]:
```suggestion
}

```

File: README.md [hunk 3]
```diff
@@ -51,8 +51,11 @@ yarn install
 
 You can then use the [Gatsby CLI commands](https://www.gatsbyjs.com/docs/reference/gatsby-cli/) `build`, `clean`, `develop` and `serve`.
 
-To build the website, run
+Our custom source plugin fetches information from GitHub, and therefore needs a GitHub [personal access token] in the environment named `GITHUB_TOKEN`. 
+
+[personal access token]: https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token
```
Comment (jdrueckert) [inline README.md:56]: Do we want to include information about how to update this in the repository settings here or is the comparable explanation in the [engine wiki](https://github.com/MovingBlocks/Terasology/wiki/Maintenance#how-to-fix-an-expired-github-action-token) sufficient if linked?

**Review by jdrueckert — DISMISSED** [review]

File: README.md (hunk 3 above)
Comment (skaldarnar) [reply README.md:56]: I think the link would be sufficient.

**Review by jdrueckert — APPROVED** [review]

**Pull request merged**